flocker/control/test/configurations/configuration_v4.py

```diff
@@ -1,11 +1,22 @@
 # Copyright ClusterHQ Inc.  See LICENSE file for details.
 
 # Generate a v4 configuration.
-# Hash to recreate: 993d92c1b6a62199cf38e45afa7d2e584d48ead1
+# Hash to recreate: 90c95c82febaceda87aed770923b284fc71f0e3f
+
+from twisted.python.filepath import FilePath
 
 from flocker.control._model import Configuration
 from flocker.control._persistence import wire_encode
-from flocker.control.test.test_persistence import TEST_DEPLOYMENT
+from flocker.control.test.test_persistence import TEST_DEPLOYMENTS
+
+_VERSION = 4
 
 if __name__ == "__main__":
-    print wire_encode(Configuration(version=4, deployment=TEST_DEPLOYMENT))
```

Comment:
:question: So the script now writes the configuration files directly. I wondered if this process might be described in our docs and need updating, but only found the instructions in the test_persistence docstrings. That's fine.